tests/e2e/specs/merchant/checkout-terms.test.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import {
+	merchant,
+	openDocumentSettingsSidebar,
+	setCheckbox,
+	unsetCheckbox,
+} from '@woocommerce/e2e-utils';
+import {
+	visitBlockPage,
+	selectBlockByName,
+	saveOrPublish,
+} from '@woocommerce/blocks-test-utils';
+
```

Comment:
```suggestion
import { createNewPost } from '@wordpress/e2e-test-utils';

```

We can import createNewPost from '@wordpress/e2e-test-utils' to create a new page.

Reply:
Sure, we could do that. But creating the post is only one out of multiple steps. That's why I created the external plugin, that you mentioned before.